extensions/wikia/ImageReview/ImageListGetter.class.php

```diff
@@ -315,4 +318,54 @@ private function getInReviewState( int $state ) : int {
 
 		else return ImageReviewStates::IN_REVIEW;
 	}
+
+	private function getLabelValues() {
+		$labels = [];
+
+		$labels['labelOk'] = [
+			'value' => ImageReviewStates::APPROVED,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)